frontend/.env.template

```
@@ -0,0 +1 @@
+baseURL = "http://localhost:5000/api"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)